### DISCUSSION TITLE
Arrows on shortcuts add

### DISCUSSION THREAD

**Discussion by D8ma** [linuxdeepin/developer-center#3280] [Features Request & Ideas | 特性请求 & 头脑风暴]
![IMG_20220630_131739.jpg](https://user-images.githubusercontent.com/107427853/176653649-e9b0c91f-1f27-45e9-ae80-a4e970c03b24.jpg)![IMG_20220630_131621.png](https://user-images.githubusercontent.com/107427853/176653664-f58902a5-31eb-4843-b860-62eed384a598.png)
![IMG_20220630_142200.png](https://user-images.githubusercontent.com/107427853/176665245-dc838d00-0db8-4ce4-8a6c-fa5c57c7966e.png)

**Comment by raspin0** [comment]
It's because those applications are not shortcuts. They are standalone .desktop files that point to application executable. It's an identical behaviour like on windows.
